QuackNBOOM/QuackNBOOM/GameObject.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  GameObject.swift
+//  QuackNBOOM
+//
+//  Created by Ngo Tuyetnhi N. and Benoit Neriah R. on 3/4/18.
+//  Copyright © 2018 Neriah and Neenee. All rights reserved.
+//
+// GameObject is the base for al other objects. This class holds only a single sprite, some
+//      attributes, and an initializer. GameObject will often be derived from but will
+//      also be used by objects with no functionality such as backgrounds.
+
+import Foundation
+import SpriteKit
+
+class GameObject {
+    var pos = CGPoint(x: 0, y: 0)                   // Position of the GameObject
```

Comment:
Many of these values are already stored on the sprite, I suggestion you subclass your GameObject from SKSprite. Alternatively you could make computed properties for these values that forward the message to the value stored in the sprite but this is needless extra work. You would gain more functionality and provide the ability to override key functions in the SKSprite class if you subclass that will greatly speed up development.